uxarray/__init__.py

```diff
@@ -1,2 +1,17 @@
 from .grid import *
 from .helpers import *
+
+
+def __version__():
```

Comment:
In my experience, it's more standard to implement `__version__` as an attribute instead of a function. 

I'd point you towards xarray's implementation here: https://github.com/pydata/xarray/blob/9909f90b4781be89e3f3ff7c87893928b3e3be6e/xarray/__init__.py#L41-L52
```python
try:
    from importlib.metadata import version as _version
except ImportError:
    # if the fallback library is missing, we are doomed.
    from importlib_metadata import version as _version

try:
    __version__ = _version("xarray")
except Exception:
    # Local copy or not installed with setuptools.
    # Disable minimum version checks on downstream libraries.
    __version__ = "999"
```

Reply:
Okay, that does make sense, thanks!